tool/tbot/main.go

```diff
@@ -117,29 +117,29 @@ func Run(args []string, stdout io.Writer) error {
 		}),
 
 		// `start` and `configure` commands
-		cli.NewLegacyCommand(startCmd, buildConfigAndStart(ctx, globalCfg)),
-		cli.NewLegacyCommand(configureCmd, buildConfigAndConfigure(ctx, globalCfg, &configureOutPath, stdout)),
+		cli.NewLegacyCommand(startCmd, buildConfigAndStart(ctx, globalCfg), cli.CommandModeStart),
+		cli.NewLegacyCommand(configureCmd, buildConfigAndConfigure(ctx, globalCfg, &configureOutPath, stdout), cli.CommandModeConfigure),
```

Comment:
I don't love introducing a new parameter here but I'm not sure we have another way to do it - command constructors need to know the use case to generate appropriate help text.

With some refactoring we could probably have `buildConfigAndStart()` and `buildConfigAndConfigure()` set the flag but that'd make the fix a lot more complex. I'm leaning toward this simpler (but verbose) solution but can tweak if desired.